### PR TITLE
Design

### PR DESCRIPTION
**Pull request by mcking49**
pulling to see if it will allow me to see changes

### CODE REVIEW

**Comment by mcking49** [comment]
This was by accident. Was meant to submit pull request to my own repo's master.

File: templates/index.html [hunk 1]
```diff
@@ -18,14 +18,49 @@
...
+</html>
+
+
+
+<!--old code, just incase it is required in future-->
+
+<!--<div class="template">
         {% block content %}
-        Welcome to the test page of <a href="http://github.com/muneeb-ali/apache-flask">apache-flask</a>.<br><br>
+        Welcome to the test page of the fucking ZOMBIE ANIMATRONICS SERVER<a href="http://github.com/muneeb-ali/apache-flask">apache-flask</a>.<br><br>
```
Comment (laynecsmall) [inline templates/index.html:57]: Ha!
Unfortunately this repo is public facing, so profanity is perhaps out of place.

Comment (laynecsmall) [inline templates/index.html:57]: So you don't want this PR reviewed and merged? 
There's a 'do not merge' option for pullrequests that revokes them.

Comment (laynecsmall) [inline templates/index.html:57]: Which I see you've found :)

Comment (mcking49) [inline templates/index.html:57]: Yeeeeaaaaahhh there may have been some serious anger and frustrations there. I'll keep it professional now on.

Correct, this SHOULD NOT be PR reviewed and merged. I forgot to change the pull request baseline thingy.

**Pull request closed without merging**